nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -2078,6 +2078,7 @@ void ddxUseMsg()
   ErrorF("-install               install colormaps directly\n");
 
   ErrorF("The NX system adds the following arguments:\n");
+  ErrorF("-options file          file containing nx/nx options\n");
```

Comment:
Can be either a file name or a plain string containing the options.

Reply:
> Can be either a file name or a plain string containing the options.

Right. Amended.